.gitignore

```
@@ -1,5 +1,5 @@
-*.yaml
-*.yml
+data.yaml
+data.yml
```

this should ignore all other yml files
```suggestion
*.yaml
*.yml
!docker-compose.dev.yml
!docker-compose.yml
```